packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeActions.ts

```diff
@@ -113,9 +114,9 @@ export class TreeActions {
     return TreeModel.getNode(this._currentModel, nodeId);
   }
 
-  public selectNode(nodeId: string, isSelected: boolean) {
+  public selectNode(nodeIds: Array<string>, changeType: SelectionChangeType) {
```

Comment:
I guess now `selectNode` should be renamed into `selectNodes` as it takes array instead of single node.